### PR TITLE
Add 52 cents gold chain

### PR DESCRIPTION
**Pull request by gkaretka**

### CODE REVIEW

**Comment by Cervator** [comment]
Hah! That's funny and clever. Nice :-)

Can confirm it works fine in-game and just needs the readme entry you mentioned in the task (https://codein.withgoogle.com/dashboard/task-instances/5713392578854912/)

**Comment by nihal111** [comment]
This looks good. Thanks! :)
Merging.

File: assets/prefabs/Equipment/Mystic/BeethovensKeyFromPiano.prefab [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "parent" : "engine:iconItem",
+    "Item" : {
+        "icon" : "Equipment:BeethovensKeyFromPiano"
+    }
```
Comment (nihal111) [inline assets/prefabs/Equipment/Mystic/BeethovensKeyFromPiano.prefab:5]: There was a missing comma after this bracket close. Fixed that myself!

Comment (gkaretka) [reply assets/prefabs/Equipment/Mystic/BeethovensKeyFromPiano.prefab:5]: Thanks, noticed today but was already fixed. Thanks

**Pull request merged**